recipe/build.sh

```diff
@@ -95,25 +91,31 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DENABLE_TESTS=ON \
       -DBUILD_UTILITIES=ON \
       -DENABLE_DOXYGEN=OFF \
-      -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
-      -DCMAKE_C_FLAGS_DEBUG=${CFLAGS} \
       -DENABLE_CDF5=ON \
-      ${CMAKE_PLATFORM_FLAGS[@]} \
       ${PARALLEL} \
+      -DENABLE_NCZARR=on \
+      -DENABLE_NCZARR_S3=off \
+      -DENABLE_NCZARR_S3_TESTS=off \
       ${SRC_DIR}
 make install -j${CPU_COUNT} ${VERBOSE_CM}
 ctest -VV --output-on-failure -j${CPU_COUNT}
```

Comment:
You need the skip here too

Reply:
I guess we need to remove that line. It seems like the result of a bad merge.